lib/PINNBench/Burgers/model.py

```diff
@@ -0,0 +1,329 @@
+    x_: shape=[num_points, 2]. First dim being x and second t.
+    u_: shape=[num_points,]
+    NUMX: number of points the in x-direction.
+    NUMT: number of points the in t-direction.
+    """
+    import matplotlib.animation as animation
+
+    u_ = u_.reshape(NUMT, NUMX).T
+    xv = x_[:,0].reshape(NUMT, NUMX)
+    tv = x_[:, 1].reshape(NUMT, NUMX)
+
+    import matplotlib.pyplot as plt
+    fig, ax = plt.subplots(2, 1)
+    ax[0].imshow(u_, interpolation='nearest',
+                      cmap='rainbow', 
+                      extent=[tv.min(), tv.max(), xv.min(), xv.max()], 
+                      origin='lower', aspect='auto' )
+    ax[0].set_xlabel('t')
+    ax[0].set_ylabel('x')
+
+    line,= ax[1].plot(xv[0], u_[:, 0])
+    text = ax[1].text(0.8, 0.9, ' ', transform=ax[1].transAxes)
+    ax[1].set_xlabel('x')
+    ax[1].set_ylabel('u')
+    def animate(t):
+        line.set_ydata(u_[:, t])
+        text.set_text('t=%.2f' % x_[:, 1][t])
+        return line, text
+    
+    ani = animation.FuncAnimation(fig, animate, frames=u_.shape[1], interval=50)
+    plt.tight_layout()
+    ani.save('./Burgers_' + name + '.gif', writer='imagemagick', fps=60)
+
+
+
+def sanityCheckData():
+    nu = 0.01         # constant in the diff. equation
+    N_u = 1000                 # number of data points in the boundaries
+    N_f = 10000               # number of collocation points
+
+    SEED = 0
+    random_state = np.random.RandomState(SEED)
+    x_ = np.linspace(-1, 1, 200)
+    t_ = np.linspace(0, 1, 100)
+    xx, tt = np.meshgrid(x_, t_)
+    x_domain = np.stack((xx.flatten(), tt.flatten()), axis=1) 
+
+    x_upper = np.ones((N_u//4, 1), dtype=float)
+    x_lower = np.ones((N_u//4, 1), dtype=float) * (-1)
+    t_zero = np.zeros((N_u//2, 1), dtype=float)
+
+    t_upper = np.random.rand(N_u//4, 1)
+    t_lower = np.random.rand(N_u//4, 1)
+    x_zero = (-1) + np.random.rand(N_u//2, 1) * (1 - (-1))
+
+    # stack uppers, lowers and zeros:
+    X_upper = np.hstack( (x_upper, t_upper) )
+    X_lower = np.hstack( (x_lower, t_lower) )
+    X_zero = np.hstack( (x_zero, t_zero) )
+
+    # each one of these three arrays haS 2 columns, 
+    # now we stack them vertically, the resulting array will also have 2 
+    # columns and 100 rows:
+    X_u_train = np.vstack( (X_upper, X_lower, X_zero) )
+
+    # shuffle X_u_train:
+    index = np.arange(0, N_u)
+    random_state.shuffle(index)
+    X_u_train = X_u_train[index, :]
+    
+    # make X_f_train:
+    X_f_train = np.zeros((N_f, 2), dtype=float)
+    for row in range(N_f):
+        x = random.uniform(-1, 1)  # x range
+        t = random.uniform( 0, 1)  # t range
+
+        X_f_train[row, 0] = x 
+        X_f_train[row, 1] = t
+
+    # add the boundary points to the collocation points:
+    X_f_train = np.vstack( (X_f_train, X_u_train) )
+
+    # make u_train
+    u_upper =  np.zeros((N_u//4, 1), dtype=float)
+    u_lower =  np.zeros((N_u//4, 1), dtype=float) 
+    u_zero = -np.sin(np.pi * x_zero)  
```

Comment:
here using `np.pi` when previously importing `pi` from `math` module, be consistent and stick to `numpy` except if particular technical requirements which should be explained in a inline comment in your code.

Reply:
Made it consistent with `numpy` and used `np.pi`.